src/auction.js

```diff
@@ -933,3 +933,38 @@ function isValidPrice(bid) {
   if (!maxBidValue || !bid.cpm) return true;
   return maxBidValue >= Number(bid.cpm);
 }
+
+/**
+ * @summary This is the function which will be called to exit module and continue the auction.
+ */
+export function continueAuction(hookConfig, delayedAuctions, callback) {
+  // only run if hasExited
+  if (!hookConfig.hasExited) {
+    // if this current auction is still fetching, remove it from the _delayedAuctions
+    const auctionIndex = delayedAuctions.findIndex(auctionConfig => auctionConfig.timer === hookConfig.timer);
+    delayedAuctions.splice(auctionIndex, 1);
+
+    // We need to know the auctionId at this time. So we will use the passed in one or generate and set it ourselves
+    hookConfig.reqBidsConfigObj.auctionId = hookConfig.reqBidsConfigObj.auctionId || generateUUID();
```

Comment:
this part is needed only by the floors module, I'd keep in there since it will be hard to figure it out if we separate it.